indico/modules/events/tracks/controllers.py

```diff
@@ -63,3 +68,39 @@ def _process(self):
                                 tracks=[{'id': int(t.getId()), 'title': to_unicode(t.getTitle())}
                                         for t in self._conf.getTrackList()])
         return jsonify_form(form)
+
+
+class RHEditTrack(RHManageTracksBase):
+
+    def _checkParams(self, params):
+        RHConferenceModifBase._checkParams(self, params)
+        self.track = Track.find_one(id=request.view_args['id'])
```

Comment:
Youdo this in two RHs -> add a `RHManageTrackBase` that contains this method and inherit from that one for edit/delete